### ISSUE TITLE
Mods being removed from sandbox.sbc

### ISSUE THREAD

**Issue by Ragonz2** [rexxar-tc/SEServerExtender#4]
As you managed to quickly fix the other issue I'm hoping you may be able to fix this that the original developer hasnt bothered to fix.
(https://github.com/SEServerExtender/SEServerExtender/issues/68)
(https://github.com/Tyrsis/SE-Community-Mod-API/issues/29)

Basically when you add mods into the sandbox.sbc file of a world without SESE it all works fine, mods are downloaded, people can join and use them.

As soon as the same server is loaded with SESE the mods in the sandbox.sbc get removed and thus the world loads without mods.

**Comment by rexxar-tc** [comment]
Well, SESE doesn't look at sandbox.sbc, it loads all the settings from SpaceEngineers-Dedicated.cfg. There isn't an elegant way to deal with multiple config files, one has to get overwritten, but how do we know which one is authoritative? We use the server cfg because changes made to sandbox.sbc while the game is running could get destroyed when the world saves. If we load settings from sandbox.sbc, any changes made in the GUI will be ignored.

I can only think of one way to manage this; I could set up SESE so that it overwrites its mod list with the one in sandbox.sbc. The problem with that is that it would be completely inappropriate for privately owned servers, I'd have to create a new branch for virtual hosts. However, it would be a fair bit of work on my end to manage two separate branches like that.

So, to sum up, this is not something that can be done in the public release. If I were to implement it, it'd have to be in a separate branch for virtual hosts. I'm not really against that idea though, I'll think about it.

Why are you modifying the sbc instead of the server config file, anyway? I presume you're using a script to take changes from your admin panel and add them to sandbox.sbc? SESE uses the same format for the config file as the vanilla server; it'd be better to modify that instead. If you use the config file, you can unify your management of SESE and vanilla servers. It shouldn't even take large changes to your script, it's just XML after all.

**Comment by tdmike** [comment]
If you add your mods in the .cfg file in the vanilla DS it also overwrites the ones in the Sanbox.sbc. It's intended that way by the dev's of SE.
